src/main/java/edu/maebe/sql2omodel/Sql2oModel.java

```diff
@@ -57,4 +87,114 @@ public boolean existJournal(UUID journal) {
         }
     }
 
+    //mood rating
```

Comment:
I think we want to split this out into two different tables. One for needs and one for mood rating